tests/algorithms/test_blurpool.py

```diff
@@ -1,96 +1,157 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""Test the blurpool algorithm.
```

Comment:
The `diffs` are a bit wrong here because I renamed `test_blurpool_algorithm.py` -> `test_blurpool.py`, and `test_blurpool.py` -> `test_blurpool_layers.py`, but github is comparing the wrong files.